ci-operator/step-registry/ipi/conf/aws/custom-vpc/ipi-conf-aws-custom-vpc-commands.sh

```diff
@@ -18,11 +18,18 @@ fi
 echo -e "subnets: $(cat ${subnet_ids_file})"
 echo -e "AZs: $(cat ${az_file})"
 
-CONFIG_PATCH="${SHARED_DIR}/install-config-subnet-azs.yaml.patch"
-cat > "${CONFIG_PATCH}" << EOF
+CONFIG_SUBNETS="${SHARED_DIR}/install-config-subnet.yaml.patch"
+cat > "${CONFIG_SUBNETS}" << EOF
 platform:
   aws:
     subnets: $(cat "${subnet_ids_file}")
+EOF
+yq-go m -x -i "${CONFIG}" "${CONFIG_SUBNETS}"
+
+
+if [[ ${ADD_ZONES} == "yes" ]]; then
```

Comment:
What case lead us have to set it as a separate parameter?

Reply:
It's a temporary workaround for 
https://github.com/openshift/release/pull/39946#issuecomment-1590836744

Reply:
So it is some bug to track?

Reply:
working on it to double check

Reply:
https://issues.redhat.com/browse/OCPBUGS-15058